grid/studies/seamstress/index.md

```diff
@@ -0,0 +1,777 @@
+---
+layout: default
+nav_exclude: true
+redirect_from: /grid-studies-seamstress/
+---
+
+# Grid Studies: seamstress
+
+seamstress is a Lua scripting environment for musical communication. It was inspired by [norns](/docs/norns) and makes use of its [scripting API](/docs/norns/reference), but it is **not** a port of the norns environment. Generally, though, seamstress is a fantastic complement to the norns scripting experience, as it has many syntactical similarities and many of the same scripting libraries.
+
+This tutorial will show the basics of interfacing with seamstresss and the grid -- including how a simple, yet immediate sequencer platform can be made with a small amount of code.
+
+## Prerequisites
+
+This study assumes a basic understanding of Lua. If you're absolutely new to Lua it may be helpful to first go through some of [the resources listed in our norns documentation](/docs/norns/studies/#learning-lua).
+
+Required:
+
+- Install seamstress on your computer: [download via GitHub](https://github.com/ryleelyman/seamstress/#installation) or `brew install seamstress`
```

Comment:
needs to be 
```
brew tap ryleelyman/seamstress
brew install seamstress
```
for now—homebrew seems to frown on people submitting their own software to core